ansible/roles/texlive/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: texlive | prereq
+  yum: name={{ item }} state=latest
+  with_items:
+    - wget
+    - perl-Tk
+    - perl-Digest-MD5
+
+- name: texlive | download and extract installer
+  unarchive:
+    copy: no
+    creates: /tmp/install-tl-unx.tar.gz
```

Comment:
@manics: in this case, the archive is going to /tmp, so may disappear in the future. Though creating dependancy on future tasks in `main.yml`, would checking for a file in `/opt/texlive/$version` make sense? Since if we already have this version, we don't need the download? (Not sure if it's bad form to couple it to the outcome of another task like that.)